genrl/deep/agents/dqn/dqn.py

```diff
@@ -0,0 +1,499 @@
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Union
+
+import gym
+import numpy as np
+import torch
+import torch.optim as opt
+from torch.autograd import Variable
+
+from genrl.deep.agents.dqn.utils import (
+    CategoricalDQNValue,
+    CategoricalDQNValueCNN,
+    DuelingDQNValueCNN,
+    DuelingDQNValueMlp,
+    NoisyDQNValue,

```

Comment:
Why is this a new file? Could you resolve this?

Seems like this has been introduced due to #198